pandas/core/reshape/merge.py

```diff
@@ -2383,12 +2380,12 @@ def renamer(x, suffix):
 
         Parameters
         ----------
-        x : original column name
+        x : original column
```

Comment:
I think the original is clearer. maybe name->label is clearer still?

Reply:
I think `name` is problematic since it doesn't have to be a string
I'll change to label